src/cli/README_SRP_SERVER.md

```diff
@@ -149,7 +149,13 @@ Done
 
 Usage: `srp server state`
 
-Print the state of the SRP server.
+Print the state of the SRP server. It could be `disabled`, `stopped` or `running`.
+
+disabled: The SRP server is not enabled yet.
+
+stopped: The SRP server is enabled but it's not active. It may become active SRP server spontaneously when detecting the lost of some active SRP server in Thread network.
+
+running: The SRP server is active and it will handle service registration.
```

Comment:
nit: bulleted list
```suggestion
- disabled: The SRP server is not enabled yet.

- stopped: The SRP server is enabled but it's not active. It may become active SRP server spontaneously when detecting the lost of some active SRP server in Thread network.

- running: The SRP server is active and it will handle service registration.
```